Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1288,11 +1286,10 @@ def prepGlyph(g, marker, index=0):
 
 def setMarkerColor(p, marker, c, cmap=None):
     # Color
-    if cmap is None:
-        if marker.colormap is not None:
-            cmap = marker.colormap
-        else:
-            cmap = 'default'
+    if marker.colormap is not None:
```

Comment:
i reviewed the code, it's good as is, if primitive has colorset then it wins. Then falls back on canvas cmap (which is passed I believed) and if nothing fallasback on vcs. Good to be merged when you're happy